narayana-full/src/main/assembly/bin.xml

```diff
@@ -202,11 +202,6 @@
 			<outputDirectory>lib/ext</outputDirectory>
 			<destName>xts_third_party_licenses.txt</destName>
 		</file>
-		<file>
```

Comment:
Is this section just cleanup or did it add something related to the `native-docker` profile?